### DISCUSSION TITLE
Claims Being Griefed With Explosion Enchantment

### DISCUSSION THREAD

**Discussion by Desjimbo** [GriefPrevention/GriefPrevention#1920] [Ideas]
Hey so I'm using the custom enchantment plugin "Ecoenchants" and they have a custom enchantment named "Dynamite". This enchantment goes on your pickaxe and when you right click with it in your hand you explode a 5x5 block radius around you. This lets people grief each others claims even though I have all explosions disabled in claims. Does anyone know anyway to fix this?

**Comment by Jikoo** [comment]
Make sure that EcoEnchants' GP hook is actually enabling. They definitely try (https://github.com/Auxilor/EcoEnchants/blob/afa143ce8feb394905778c11be953292df0f9d25/eco-core/core-plugin/src/main/java/com/willfp/ecoenchants/enchantments/ecoenchants/spell/Dynamite.java#L66 and https://github.com/Auxilor/eco/blob/a526f517809f7e1d886f6483a47eed77cf1829e7/eco-core/core-plugin/src/main/kotlin/com/willfp/eco/internal/spigot/integrations/antigrief/AntigriefGriefPrevention.kt#L12) to respect claims, but if the hook isn't registered claims won't be respected.

Really they should probably just fire a BlockBreakEvent instead of running specific protection plugin integrations, seems like a compatibility nightmare. Either way, you'll need to contact them about this. In the mean time, I believe all their enchantments can be disabled individually in the config, would recommend you disable that one.
